### PR TITLE
Updates README with method signatures + helper methods for setting fromUri

### PR DESCRIPTION
**Pull request by jmelberg-okta**
- Updates README on all libs to include customizing the `fromUri` for after-auth navigation.
- Updates Angular README to include all exposed method signatures

### CODE REVIEW

File: packages/okta-react/src/Auth.js [hunk 1]
```diff
@@ -68,8 +68,12 @@ export default class Auth {
     return accessToken ? accessToken.accessToken : undefined;
   }
 
+  setFromUri(uri) {
+    localStorage.setItem('referrerPath', uri);
+  }
+
   async login() {
-    localStorage.setItem('secureRouterReferrerPath', this._history.location.pathname);
+    this.setFromUri(this._history.location.pathname)
```
Comment (jmelberg-okta) [inline packages/okta-react/src/Auth.js]: Modifying this value to be consistent across all libraries

File: packages/okta-angular/README.md [hunk 2]
```diff
@@ -101,13 +101,40 @@ Finally, import the `OktaAuthModule` into your `NgModule`, and instantiate it by
...
 
-### Reference
-**Configuration Options**
-  - `issuer` **(required)**: The OpenID Connect `issuer`
-  - `clientId` **(required)**: The OpenID Connect `client_id`
-  - `redirectUri` **(required)**: Where the callback is hosted
-  - `scope` *(optional)*: Reserved or custom claims to be returned in the tokens
-  - `onAuthRequired` *(optional)*: Accepts a callback to make a decision when authentication is required. If not supplied, `okta-angular` will redirect directly to Okta for authentication.
+## Use the Access Token
+When your users are authenticated, your Vue application has an access token that was issued by your Okta Authorization server. You can use this token to authenticate requests for resources on your server or API. As a hypothetical example, let's say you have an API that provides messages for a user. You could create a `MessageList` component that gets the access token and uses it to make an authenticated request to your server.
```
Comment (mraible) [inline packages/okta-angular/README.md]: Vue > Angular

File: packages/okta-angular/README.md [hunk 3]
```diff
@@ -101,13 +101,40 @@ Finally, import the `OktaAuthModule` into your `NgModule`, and instantiate it by
...
+    // Make request
+    this.http.get(
+      'http://localhost:{serverPort}/api/messages',
+      new RequestOptions({ headers: headers })
+    )
+    .map(res => res.json())
+    .subscribe((messages: Array<Object>) => messages.forEach(message => this.messages.push(message)));
+  }
+}
+```
```
Comment (mraible) [inline packages/okta-angular/README.md]: You could also use an HttpInterceptor with Angular 4.3+ and HttpClient.

File: packages/okta-angular/src/okta/okta.service.ts [hunk 4]
```diff
@@ -88,6 +88,14 @@ export class OktaAuthService {
       return this.oktaAuth.tokenManager.get('idToken');
     }
 
+    /**
+     * Returns the user context from the userinfo endpoint.
+     */
+    async getUser() {
+      const accessToken = this.oktaAuth.tokenManager.get('accessToken');
+      return accessToken ? this.oktaAuth.token.getUserInfo(accessToken) : undefined;
```
Comment (mraible) [inline packages/okta-angular/src/okta/okta.service.ts]: This is awesome! 👏

File: packages/okta-angular/README.md (hunk 3 above)
Comment (jmelberg-okta) [reply packages/okta-angular/README.md]: Great suggestion! I want to keep this consistent with our [quickstart](https://developer.okta.com/quickstart/#/angular/nodejs/generic) guide for now, but if the community is using `HttpClient` over the `Http` module, we should update both of our docs.

Just out of curiosity, is `HttpClient` the favorable choice?

Comment (mraible) [reply packages/okta-angular/README.md]: Yes, `Http` is deprecated in Angular 5.

**Pull request closed without merging**